core-bundle/src/Command/AbstractLockedCommand.php

```diff
@@ -19,14 +19,14 @@
 use Symfony\Component\Lock\Store\FlockStore;
 
 /**
- * @deprecated Deprecated since Contao 4.7, to be removed in Contao 5.0; use
- *             the lock service instead
+ * @deprecated Deprecated since Contao 4.7, to be removed in Contao 5.0; Use
+ *             the Symfony Lock component instead.
```

Comment:
```suggestion
 * @deprecated Deprecated since Contao 4.7, to be removed in Contao 5.0; use
 *             the Symfony Lock component instead
```

There is no capitalization after a semicolon IMHO.

Reply:
right, should be in upper-case - I've missed it, because the 
sentence hat a point instead of a semicolon.